src/main/steps/applicant1/hub-page/template.njk

```diff
@@ -17,54 +16,14 @@
       <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible govuk-section-break--thinner">
       <div class="govuk-!-padding-bottom-8"></div>
 
-      {{ hmctsProgressBar({
-        classes: 'hub-progress-bar',
-        items: [{
-          label: {
-            text: applicationSubmitted
-          },
-            complete: progressionIndex == 0,
-            active: progressionIndex == 0
-          }, {
-            label: {
-              text: conditionalOrderApplication
-            },
-            complete: progressionIndex > 0,
-            active: hasApplicantConfirmedReceipt
-          }, {
-            label: {
-              text: conditionalOrderGranted
-            },
-            complete: progressionIndex > 1,
-            active: progressionIndex == 1
-          }, {
-            label: {
-              text: finalOrderApplication
-            },
-            complete: progressionIndex >= 2,
-            active: progressionIndex == 2
-          }, {
-            label: {
-              text: applicationEnded
-            },
-            complete: progressionIndex >= 3,
-            active: progressionIndex == 3
-          }]
-          }) }}
+      {% include "./progress-bar.njk" %}
 
       <h2 class="govuk-heading-m">{{ subHeading1 }}</h2>
       <div class="govuk-panel govuk-panel--outstanding">
-        {% if progressionIndex == 0 and not hasApplicantConfirmedReceipt %}
-          {% for property, line in applicationSubmittedLatestUpdate %}
-            <p class="govuk-body">{{ line }}</p>
-          {% endfor %}
-          {% block form %}
-            {% include "./form.njk" %}
-          {% endblock %}
-          {% else %}
-          {% for property, line in applicantConfirmedReceiptLatestUpdate %}
-            <p class="govuk-body">{{ line }}</p>
-          {% endfor %}
+        {% if isJointApplication %}
+          {% include "./joint/hub-1.njk" %}
+        {% else %}
+          {% include "./sole/hub-1.njk" %}
```

Comment:
can't these just be "./joint/hub.njk" and "./sole/hub.njk", unless you think we should use different templates for the different hub versions?

Reply:
ill change it as thats fine for now, but as theres multiple pages of the hub, there will be different templates or one template with multiple if statements